code/game/objects/effects/portals.dm

```diff
@@ -17,6 +17,7 @@
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "portal"
 	anchored = TRUE
```

Comment:
Despite being anchored they do move in zero-g, is this intended?

Reply:
no

Reply:
okay I will add a fix for that too